themes/src/main/resources/theme/base/admin/messages/admin-messages_ru.properties

```diff
@@ -1,915 +1,1267 @@
-headers=\u0417\u0430\u0433\u043E\u043B\u043E\u0432\u043A\u0438
-brute-force-detection=\u041E\u043F\u0440\u0435\u0434\u0435\u043B\u0435\u043D\u0438\u0435 Brute Force
+realm-detail.enabled.tooltip=Пользователи и клиенты могут получить доступ к Realm только если он включен
+realm-detail.oidc-endpoints.tooltip=Показать конфигурацию конечных точек OpenID Connect
+registrationAllowed=Регистрация пользователя
+registrationAllowed.tooltip=Включить/выключить страницу регистрации. Ссылка для регистрации будет также показана на форме входа.
+registrationEmailAsUsername=Email как имя пользователя
+registrationEmailAsUsername.tooltip=Если включено, то на форме регистрации поле имени пользователя будет скрыто и в качестве имени пользователя для новых пользователей будет использоваться email.
+editUsernameAllowed=Редактируемое имя пользователя
+editUsernameAllowed.tooltip=Если включено,то имя пользователя можно будет отредактировать, иначе оно будет доступным только для чтения.
+resetPasswordAllowed=Забыли пароль
+resetPasswordAllowed.tooltip=Показывает ссылку на странице входа для пользователя, по переходу на которую пользователь сможет восстановить свои данные для входа.
+rememberMe=Запомнить меня
+rememberMe.tooltip=Показать чекбокс на странице входа, чтобы разрешить пользователю запомнить вход в учетную запись в случае если браузерная сессия устареет.
+loginWithEmailAllowed=Вход по email
+loginWithEmailAllowed.tooltip=Разрешает пользователям входить с их email.
+duplicateEmailsAllowed=Дублирующиеся email
+duplicateEmailsAllowed.tooltip=Разрешает разным пользователям иметь один и тот же email. Изменение этой настройки также очистит пользовательский кэш. Рекомендуется ручное обновления констрейнтов email для существующих пользователей в базе данных после выключения поддержки дублирующихся email.
```

Comment:
because it database term, may be we should not translate this word? Or translate it as "ограничения"? I propose next variant "После выключения поддержки дублирующихся email рекомендуется вручную почистить в базе данных ограничения по email существующим пользователям."